lib/katello/tasks/clean_published_repo_directories.rake

```diff
@@ -0,0 +1,54 @@
+namespace :katello do
+  task :clean_published_repo_directories => ['environment'] do
```

Comment:
I would add a description here explaining what it does and explaining the COMMIT option